documentation/tutorials/kw-env.rst

```diff
@@ -64,3 +64,32 @@ Let's say that you want to switch from one config to another, you can just use::
 If you want to check if everything looks correct, you can use::
 
  kw vars
+
+How to customize your bash terminal to display the current environment
+----------------------------------------------------------------------
+
+This feature allows you to view the current environment in your bash terminal.
+It is especially useful when working with multiple environments within kw, as it
+helps you stay aware of the context you are working in, preventing errors and confusion.
+
+To enable this functionality, follow these steps:
+
+Run the kworkflow installation command::
+
+  ./setup -i
+
+If you have already installed kworkflow, proceed to the next step.
```

Comment:
Maybe:

If you did not install kw yet, run the below command; otherwise, go to the next step.